RealmTasks Android/app/src/main/java/io/realm/realmtasks/list/TaskAdapter.java

```diff
@@ -41,12 +44,29 @@ public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
         if (task.isValid()) {
```

Comment:
Not sure, I didn't change this.  Can we open this as a separate investigation task/issue?  It's not related to this PR.